src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -597,16 +597,54 @@ public override async Task<TResponse> InvokeMethodGrpcAsync<TResponse>(string ap
 
         #region State Apis
 
+        /// <inheritdoc />
         public override async Task<IReadOnlyList<BulkStateItem>> GetBulkStateAsync(string storeName, IReadOnlyList<string> keys, int? parallelism, IReadOnlyDictionary<string, string> metadata = default, CancellationToken cancellationToken = default)
+        {
+            var rawBulkState = await GetBulkStateRawAsync(storeName, keys, parallelism, metadata, cancellationToken);
+
+            var bulkResponse = new List<BulkStateItem>();
+            foreach (var item in rawBulkState)
+            {
+                bulkResponse.Add(new BulkStateItem(item.Key, item.Value.ToStringUtf8(), item.Etag));
+            }
+
+            return bulkResponse;
+        }
+        
+        /// <inheritdoc/> 
+        public override async Task<IReadOnlyList<BulkStateItem<TValue>>> GetBulkStateAsync<TValue>(string storeName,
+            IReadOnlyList<string> keys, int? parallelism, IReadOnlyDictionary<string, string> metadata = default,
+            CancellationToken cancellationToken = default)
```

Comment:
Nit: If we've started to separate the parameters onto separate lines, our convention is to have each parameter on their own line.

Reply:
I've updated my editor settings so it honors this going forward. Corrections made in latest commit.